website_product_show_uom/README.rst

```diff
@@ -0,0 +1,101 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========
+Show UoM
+=========
+This module brings the users the possibility to:
+
+1. Show/hide UoM on product list page
+2. Show/hide UoM on all product pages
+3. Show/hide UoM on checkout page
+4. Show/hide UoM on payment page
+
+Usage
+=====
+
+**Show UoM on product list page**
+
+1. Go to website
+2. Open shop menu
+3. Click **Customize** menu
+4. Toggle **Show UoM** option to show/hide UoM
+
+UoM will show after product price
+
+**Show UoM on each product page(s)**
+
+1. Go to website
+2. Open shop menu
+3. Open one product page
+4. Click **Customize** menu
+5. Toggle **Show UoM** option to show/hide UoM
+
+UoM will show after product price
+
+**Show UoM on checkout page**
+
+1. Go to website
+2. Open shop menu
+3. Open one product page
+4. Click **Add to chart** button. This will open checkout page
+5. Click **Customize** menu
+6. Toggle **Show UoM** option to show/hide UoM
+
+New column for UoM will show after quantity on Shopping Chart table
```

Comment:
A new column for UoM will be displayed after the quantity on Shopping Chart table.